gui/system/forms.py

```diff
@@ -2083,32 +2090,31 @@ def _types_avail(cls, disks):
     @staticmethod
     def _grp_type(num):
         check = OrderedDict((
-            ('mirror', lambda y: y == 2),
+            ('MIRROR', lambda y: y == 2),
             (
-                'raidz',
+                'RAIDZ1',
                 lambda y: False if y < 3 else math.log(y - 1, 2) % 1 == 0
             ),
             (
-                'raidz2',
+                'RAIDZ2',
                 lambda y: False if y < 4 else math.log(y - 2, 2) % 1 == 0
             ),
             (
-                'raidz3',
+                'RAIDZ3',
                 lambda y: False if y < 5 else math.log(y - 3, 2) % 1 == 0
             ),
-            ('stripe', lambda y: True),
+            ('STRIPE', lambda y: True),
         ))
         for name, func in list(check.items()):
             if func(num):
                 return name
-        return 'stripe'
+        return 'STRIPE'
 
     @classmethod
     def _grp_autoselect(cls, disks):
 
         higher = cls._higher_disks_group(disks)
-        groups = OrderedDict()
-        grpid = 0
+        groups = defaultdict(list)
```

Comment:
We don't want an OrderedDict anymore?

Reply:
No, order is not important anymore in new API